mac

```diff
@@ -126,13 +127,16 @@ brew "rbenv"
 brew "nvm"
 brew "pyenv"
 brew "pipenv"
+brew "tfenv"
```

Comment:
We're not using terraform though right?

Reply:
Or rbenv, or gpg-soute

Reply:
Or redis

Reply:
`tfenv`: Not yet, though I think there's a good chance we will. 
`gpg-suite`: I can't recall for sure, but I think the libs included in this are used for verifying SHAs on downloaded packages (though I don't often do that).
`redis`: Not yet. May come into play when we start having user sessions in Django.

Reply:
I'd rather limit the scope of things we're installing to things that are part of the "recommended" flow for now. Otherwise, I highly recommend the following:
```
hub
jq
```